toolchain/lower/context.cpp

```diff
@@ -41,7 +41,21 @@ auto Context::GetFileContext(const SemIR::File* file,
   return insert_result.value().get();
 }
 
+auto Context::LowerPendingDefinitions() -> void {
+  // Lower function definitions for generics.
+  // This cannot be a range-based loop, as new definitions can be added
+  // while building other definitions.
+  // NOLINTNEXTLINE
```

Comment:
```suggestion
  // NOLINTNEXTLINE(modernize-loop-convert)
```

Guessing what this NOLINT is for, would be good to name the check

Reply:
I guess that's probably right. (I didn't add this NOLINTNEXTLINE, it's just been moved from file_context.cpp). Let's see if clang-tidy is happy :)